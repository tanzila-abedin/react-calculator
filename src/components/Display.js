import PropTypes from 'prop-types';
// import { Textfit } from 'react-textfit';
import '../style/App.css';

const Display = ({ value }) => <div className="display">{value || '0'}</div>;

Display.propTypes = {
  value: PropTypes.string,
};

Display.defaultProps = {
  value: 0,
};
export default Display;
