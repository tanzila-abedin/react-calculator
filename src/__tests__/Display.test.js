import renderer from 'react-test-renderer';
import Display from '../components/Display';

test(' Snapshot of Display component to show output of a button when clicked', () => {
  const display = renderer.create(<Display value="10" />).toJSON();
  expect(display).toMatchSnapshot();
});
