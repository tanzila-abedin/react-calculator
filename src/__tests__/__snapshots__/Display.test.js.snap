// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Snapshot of Display component to show output of a button when clicked 1`] = `
<div
  className="display"
>
  10
</div>
`;
